airbyte-db/src/main/java/io/airbyte/db/Databases.java

```diff
@@ -41,6 +41,10 @@ public static JdbcDatabase createRedshiftDatabase(String username, String passwo
     return createJdbcDatabase(username, password, jdbcConnectionString, "com.amazon.redshift.jdbc.Driver");
   }
 
+  public static Database createSqlServerDatabase(String username, String password, String jdbcConnectionString) {
+    return createDatabase(username, password, jdbcConnectionString, "com.microsoft.sqlserver.jdbc.SQLServerDriver", SQLDialect.DEFAULT);
+  }
```

Comment:
`createPostgresDatabase` is used by the Scheduler to create the metadata database, but RedShift is never used and the SQL Server is only using this function in tests where can easily be changed by `createDatabase` function, maybe we can remove them?

Reply:
This came from getting the tests working too.  I don't see any particular problem with having a few wrapper functions around to simplify things.